tests/cypress/e2e/unit_tests/advanced_filtering.spec.ts

```diff
@@ -16,53 +16,56 @@ import { TopLevelMenu } from '~/cypress/support/toplevelmenu';
 import '~/cypress/support/functions';
 import { Elemental } from '~/cypress/support/elemental';
 import { contains } from 'cypress/types/jquery';
+import filterTests from '~/cypress/support/filterTests.js';
```

Comment:
Maybe more coherent to do the import like this (without the `.js`), no?
```javascript
import filterTests from '~/cypress/support/filterTests';
```

Reply:
I tried it one more time and looks like `.js` must be used because it is imported inside a `.ts` file.